ykllvmwrap/src/ykllvmwrap.cc

```diff
@@ -119,7 +120,7 @@ extern "C" void *__ykllvmwrap_irtrace_compile(char *FuncNames[], size_t BBs[],
   llvm::Function *DstFunc = llvm::Function::Create(
       FType, Function::InternalLinkage, TraceName, DstMod);
   DstFunc->setCallingConv(CallingConv::C);
-  auto DstBB = BasicBlock::Create(Context, "bb0", DstFunc);
+  auto DstBB = BasicBlock::Create(Context, ENTRY_BLOCK, DstFunc);
```

Comment:
I think we could also not give it a name at all by passing `""`. I'm not sure how much more helpful the name for debugging is anyway since the entry block isn't really hard to find.

Reply:
The second commit is just for sake of convention. As Laurie said to me last week: "when in Rome" (buy a good sun hat).

Reply:
Not sure what you are saying. I'm arguing that we don't really need to pass anything here. If this was really just a convention fix, then you shouldn't have changed `bb0` to `entry` either.

Reply:
You are right that we don't have to name the block. I only do because I see it named "entry" everywhere. E.g. https://llvm.org/docs/tutorial/MyFirstLanguageFrontend/LangImpl04.html

Reply:
The few times I inspected LLVM IR, the first block never seemed to have a name.

Anyway, I just wanted to point it out. The decision is ultimately yours.